File: DataProcess_People/src/main/scala/buct/jk1702/Bean/CovidBean.scala
package buct.jk1702.Bean

/**
 * @author Zednight
 * @date 2020/10/6 16:45
 */
case class CovidBean(
                      provinceName: String , //省份名称

                      provinceShortName: String , //省份短名

                      cityName: String ,
                      currentConfirmedCount: Int , //当前确诊人数

                      confirmedCount: Int , //累计确诊人数

                      suspectedCount: Int , //疑似病例人数

                      curedCount: Int , //治愈人数

                      deadCount: Int , //死亡人数

                      locationId: String , //位置id

                      pid: String ,
                      statisticsData: String , //每一天的统计数据

                      cities: String , //下属城市

                      datetime: String //获取时间
                    )

File: DataAcquisition/src/main/java/buct/jk1702/Entity/CovidBean.java
package buct.jk1702.Entity;


import lombok.Data;

@Data
public class CovidBean {
    private String provinceName;//省份名称
    private String provinceShortName;//省份短名
    private String cityName;
    private Integer currentConfirmedCount;//当前确诊人数
    private Integer confirmedCount;//累记确诊人数
    private Integer suspectedCount;//疑似病例人数
    private Integer curedCount;//治愈人数
    private Integer deadCount;//死亡人数
    private String locationId;//位置id
    private String pid;//位置id
    private String statisticsData;//每一天的统计数据
    private String cities;//下属城市
    private String datetime;//获取时间
}
